packages/front-end/components/Layout/Layout.tsx

```diff
@@ -192,8 +193,26 @@ const backgroundShade = (color: string) => {
 
 const Layout = (): React.ReactElement => {
   const [open, setOpen] = useState(false);
+  const [isOldVersion, setIsOldVersion] = useState(false);
   const settings = useOrgSettings();
 
+  const build = getGrowthBookBuild();
+
+  useEffect(() => {
+    async function checkVersions() {
+      const res = await fetch(
+        "https://api.github.com/repos/growthbook/growthbook/branches/main"
+      );
+      const jsonRes = await res.json();
+      const remoteSha = jsonRes.commit.sha;
+
+      if (build.sha !== remoteSha) setIsOldVersion(true);
```

Comment:
We should wait until the latest commit is at least 20 minutes old, so it has time to go through the build process. Otherwise, we may prompt people to upgrade before the new docker version is available.

Reply:
Also, add a sanity check to make sure `build.sha` is set first. We don't want this showing up for people running yarn dev and missing the buildinfo folder.